app/src/main/java/org/fossasia/openevent/general/event/EventsViewModel.kt

```diff
@@ -2,16 +2,21 @@ package org.fossasia.openevent.general.event
 
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
+import android.view.View
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.content_no_internet.view.*
+import kotlinx.android.synthetic.main.fragment_events.view.*
+import org.fossasia.openevent.general.data.Network
 import org.fossasia.openevent.general.data.Preference
 import timber.log.Timber
 
 class EventsViewModel(private val eventService: EventService, private val preference: Preference) : ViewModel() {
 
     private val compositeDisposable = CompositeDisposable()
     private val tokenKey = "LOCATION"
+    private val networkData = Network()
```

Comment:
Inject instead of initializing

Reply:
Please give me a hint how to Inject Network class?